CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 
 ### Bugfixes
+- [#1335](https://github.com/openscope/openscope/issues/1329) - Fix inconsistent state of control buttons when window loses and regains focus.
```

Comment:
please add a new line after that so git will understand that you add a line instead of replacing one.

Reply:
Hm? I did add rather than replace a line. Compare [before](https://raw.githubusercontent.com/openscope/openscope/3763f6e966350690699bea6f00a21147c0e3490b/CHANGELOG.md) vs [after](https://raw.githubusercontent.com/openscope/openscope/6da1f0ad6a2d3168a90cd3a206930296fb6917ec/CHANGELOG.md).  
Or is there something else I'm missing? (Besides the incorrect link -- that I will fix)

Reply:
My mistake. I got confused by the red minus...